app/utils/hentFraApi.ts

```diff
@@ -1,20 +1,19 @@
-import søkerMock from '~/mock/søkerMock';
 import { fetchWithToken } from '~/server/authorization';
 import { EMiljø } from '~/typer/miljø';
 import { ISøker } from '~/typer/søker';
+import { Session } from '@remix-run/node';
 
 const STI: string = '/api/oppslag/soker';
 const LOKAL_URL_BACKEND: string = 'http://localhost:8099' + STI;
 const API_URL_BACKEND: string =
-  'https://nav-familie-endringsmelding-api.fly.dev/' + STI;
+  'https://nav-familie-endringsmelding-api.fly.dev' + STI;
 
-export const hentSøker = async (request: Request): Promise<ISøker> => {
+export const hentSøker = async (session: Session): Promise<ISøker> => {
   switch (process.env.ENV) {
     case EMiljø.LOKAL:
-      return (await fetchWithToken(request, LOKAL_URL_BACKEND)).json();
+      return (await fetchWithToken(session, LOKAL_URL_BACKEND)).json();
     case EMiljø.PRODUKSJON:
-      return (await fetchWithToken(request, API_URL_BACKEND)).json();
     default:
-      return Promise.resolve(søkerMock);
+      return (await fetchWithToken(session, API_URL_BACKEND)).json();
   }
```

Comment:
Da slutter vi nå å mocke? Man henter alltid fra backend når vi jobber og ikkje spinner opp APIet?

Reply:
Nå vil vi gå mot backend dersom miljø ikke er satt. Det kan hende at ønsker å gjøre noe annet. Jeg gjorde det egentlig bare sånn nå for å få det til å funke 😊

Reply:
Det gir meining det ☀️
Tenkte kanskje det var dumt å gjør mange kall til backend ( i forhold til resurser og kostnader. På universitetet mocket vi en del, men det er kanskje fordi alle brukte samme backend på prosjektet og serveren var eid/betalt av NTNU😅)

Reply:
Skjønner! Dersom vi mocker blir det vanskeligere å oppdage om noe er feil. I NAV prøver vi å holde miljøene våre så like som de skal være når de er i bruk som mulig slik at vi kan oppdage feil tidlig 😊

Reply:
Egentlig ganske smart :)